packages/core/src/main/logger/ipc-logging-listener.injectable.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import ipcFileLoggerInjectable from "./ipc-file-logger.injectable";
+import { getMessageChannelListenerInjectable } from "../../common/utils/channel/message-channel-listener-injection-token";
+import type {
+  IpcFileLogObject } from "../../common/logger/ipc-file-logger-channel";
```

Comment:
nit: this formatting looks really weird.

(If it is a prettier issue, disregard)

Reply:
That's a result of running `lint:fix` to please the PR checks, agreed that it does not look ok. Prettier seems to make it nice though.

I've also recently had some issues with VS Code not showing linting issues as the PR github actions.